### ISSUE TITLE
Unsafe code may only appear if compiling with /unsafe (HDF5.NET.Benchmarks)

### ISSUE THREAD

**Issue by LiorBanai** [Apollo3zehn/HDF5.NET#24]
I get the following error when try to build the solution:
![image](https://user-images.githubusercontent.com/36262995/174348300-cd9996f2-b677-42bf-87c3-8d5f12bc81aa.png)

**Comment by Apollo3zehn** [comment]
You can add

```
  <PropertyGroup>
    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
  </PropertyGroup>
```

 to your csproj file to solve this. I will check it Al's next week.

**Comment by LiorBanai** [comment]
I just unloaded the project but assumed you have the same issue.

**Comment by Apollo3zehn** [comment]
Solved in commit 0f311b2cc8c097ce0fe5340224d471f0dac265d6.
